### DISCUSSION TITLE
Why Chia daemon eat 190 GB of RAM?

### DISCUSSION THREAD

**Discussion by chiaXXL** [Chia-Network/chia-blockchain#2866] [General]
Just test new x2 CPU Xeon server,

Started 29 parallel plots
and as usual chia wallet stuck and get white ant 33% 
Im tired with this whats wrong with windows chia wallet? I have tested it on so many configurations and everywhere it is crashed if use heavy load.  Of course the configuration of hardware is enough.
Here is my plotting server:
https://prnt.sc/124hkhh
here is proof of memory eating by chia daemon:
https://prnt.sc/124hm7g

I dont know what to do else. The windows is 2019 server fresh install from MSDN.

It seem that windows plotting is not designed for heavy load. Im ok with this. But can you fcn put the warning into FAQ???
For heavy parallel plotting you nee linux!
Im mad. 4 years of development? Why do you offer windows client. You should remove it or at least warn people. Thats not fair.

**Comment by cccat6** [comment]
Even I have to use Windows for plotting. I will use CLI but not GUI. It is really inconvenient to use GUI for plotting since there are lots of BUGs right now.
Try to write a script for plotting and it will make your life much easier.

**Comment by nomadengineer** [comment]
At the end of plotting 15 plots. daemon is at a really high memory usage and the chia gui is blank!

![image](https://user-images.githubusercontent.com/45670423/116827211-51827f80-abb1-11eb-86be-5c0131c8976f.png)

**Comment by seeden** [comment]
Please try to use the latest version 1.1.3. It should be fixed
